components/expenses/graphql/queries.ts

```diff
@@ -15,6 +15,9 @@ export const expensePageQuery = gql`
     expense(expense: { legacyId: $legacyExpenseId }, draftKey: $draftKey) {
       id
       ...ExpensePageExpenseFields
+      permissions {
+        canDeclineExpenseInvite(draftKey: $draftKey)
+      }
```

Comment:
:ice_cream:  As an alternative if we don't to force clients to provide the `draftKey` again (since it's already passed to the parent expense, we can use the `allowContextPermission` mechanism in the API:

```ts
// Expense resolver
if (hasValidDraftKey) {
  allowContextPermission(req, PERMISSION_TYPE.DECLINE_EXPENSE_INVITATION, expense.id);
}
```

```ts
// canDeclineExpenseInvite resolver
if (getContextPermission(req, PERMISSION_TYPE.DECLINE_EXPENSE_INVITATION, expense.id)) {
  return true;
}
```